### ISSUE TITLE
USB drive do not appear in the loader

### ISSUE THREAD

**Issue by PCasey212** [abdyfranco/next-loader#18]
Xserve 3.1. Successfully installed the Next Loader as the "master" boot loader of my system. OSX 10.11.6 load flawlessly.

Problem : the USB drive are no more bootable on the system. They do not appear on the loader. Have tried the 3 usb ports without success.

At this time, my OSX El Capitan install drive isn't recognize. Is it normal with Xserve or I failed something ?

**Comment by kumowoon1025** [comment]
I think you might have to build it yourself for an Xserve... Some drivers work because Xserve3,1 is similar to MacPro5,1 I'm assuming but I'm also having trouble getting the drivers to load and mount everything

**Comment by PCasey212** [comment]
Hi kumowwon1025,

Thank you for your reply. Since there is not help from the devs and this projects is simply dead, I removed the loader from my Xserve and continu to work with the normal Apple stuff.

Parallel isn't the best way but it is a working way...

**Comment by abdyfranco** [comment]
I installed an Xserve 2,1 on my Home Lab a week ago, I will check tomorrow if recognizes my macOS Sierra installation drive.

**Comment by PCasey212** [comment]
Thank you for the answer. Keep me in touch !

**Comment by josch1710** [comment]
Are there any progress on this issues. I really want to UEFI boot windows and I need the USB Drive for that.

**Comment by kumowoon1025** [comment]
@josch1710 Mac Pro? There shouldn't be any issues, unless you meant Xserve?

**Comment by josch1710** [comment]
No, no. I know what I wrote: USB sticks, that would show up in the Mac boot screen wouldn't in NextLoader.

**Comment by kumowoon1025** [comment]
@josch1710 Sorry for doubting you then, it must be because of the different chipset in a MacPro 3,1 than 5,1 (which the release is compiled for). Compiling on your own specifically to support the Mac Pro 3,1 with appropriate efi drivers should work better.
